Lombiq.Tests.UI.Samples/Tests/BasicVisualVerificationTests.cs

```diff
@@ -33,18 +33,19 @@ public BasicVisualVerificationTests(ITestOutputHelper testOutputHelper)
                 context.SetViewportSize(CommonDisplayResolutions.HdPlus);
 
                 var navbarElementSelector = By.ClassName("navbar-brand");
-                // We set the browser's window size, DPI, and scale settings explicitly to make the test environment
-                // similar on every platform, but we have no control over the fonts. In this case, the selected block
-                // element size, mainly height, depends on the font used on a given platform.
-                // We don't know the exact size of the selected element, so we should select the interesting region for
-                // reference and captured it as a bitmap and use it at the end. Without cropping the reference and
-                // captured bitmap to the same size, ImageSharpCompare.CalcDiff(...) throws an
-                // ImageSharpCompareException.
-                var cropRegion = new Rectangle(0, 0, 1400, 23);
 
-                // Here we check that the rendered content visually equals the reference image within a given error
-                // percentage. You can read more about this in the AssertVisualVerificationApproved method documentation.
-                context.AssertVisualVerificationApproved(navbarElementSelector, 6, cropRegion);
+                // Here we check that the rendered content visually equals the baseline image within a given error
+                // percentage only for the platforms given by WithPlatforms() configuration. You can read more about
+                // this in the AssertVisualVerificationApproved method documentation.
+                context.AssertVisualVerificationApproved(navbarElementSelector, 0, configurator: configuration => configuration
+                // With this configuration, we select the platforms to which the verification is applied.
+                .WithPlatforms(PlatformID.Win32NT, PlatformID.Unix)
+                // With this configuration, the PlatformID will be used as a suffix at the end of the baseline image
+                // file name.
+                .WithUsePlatformAsSuffix()
+                // With this configuration, the browser name provided by selenium will be used as a suffix at the end of
+                // the baseline image file name.
+                .WithUseBrowserNameAsSuffix());
```

Comment:
Similar indentation in `VerifyHomePage` too:

```suggestion
                context.AssertVisualVerificationApproved(
                    navbarElementSelector,
                    0,
                    configurator: configuration =>
                        configuration
                            // With this configuration, we select the platforms to which the verification is applied to.
                            .WithPlatforms(PlatformID.Win32NT, PlatformID.Unix)
                            // With this configuration, the PlatformID will be used as a suffix at the end of the
                            // baseline image file name.
                            .WithUsePlatformAsSuffix()
                            // With this configuration, the browser name provided by Selenium will be used as a suffix
                            // at the end of the baseline image file name.
                            .WithUseBrowserNameAsSuffix());
```